packages/plugin-ext/src/hosted/node/plugin-reader.ts

```diff
@@ -58,7 +58,8 @@ export class HostedPluginReader implements BackendApplicationContribution {
                         // the request was already closed
                         return;
                     }
-                    if ('code' in e && e['code'] === 'ENOENT') {
+                    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                    if ((e as any).code === 'ENOENT') {
```

Comment:
If we properly type `e` based on what we might expect then we would not need to cast as `any`?

```ts
res.sendFile(filePath, { root: localPath }, e => {
```

to

```ts
res.sendFile(filePath, { root: localPath }, (e: Error & { code?: string }) => {
```

Reply:
`any` is more convenient here, we still end up lying to the compiler (can make a typo either in the `{ code?: string }` or `(e as any).code`).

Reply:
Found `NodeJS.ErrnoException` that also inherits `Error`, will use that instead.